components/automate-cli/cmd/chef-automate/verify_test.go

```diff
@@ -14,710 +13,6 @@ const (
 	parseConfigFileError = "Unable to parse config file"
 )
 
-func TestVerifyCmdFunc(t *testing.T) {
```

Comment:
Please update the test cases, don't remove them as coverage on verify file is decreasing as compared to main branch

Reply:
It will be taken care in upcoming issue, it's not in scope of this pr.